### ISSUE TITLE
todo: find score classification/demarcation thresholds

### ISSUE THREAD

**Issue by nkhalsa** [rkhaled98/Karmalutional-Network#3]
The output layer of the network will likely be a softmax classifier, which will likely use five categories. These will correspond to "terrible," "bad," "neutral," "good," and "top." The plan is to label the training data as such based on comment scores. It may be challenging to find appropriate comment score thresholds for demarcation without enough humans to classify comments into those categories. Only having one person determine the score could result in a network training to a single person's ideas of what a good comment is.

**Comment by rkhaled98** [comment]
Placeholder thresholds will be used for the meantime.
